x/programs/examples/imports/hashmap/hashmap.go

```diff
@@ -0,0 +1,153 @@
+// Copyright (C) 2023, Ava Labs, Inc. All rights reserved.
+// See the file LICENSE for licensing terms.
+
+package hashmap
+
+import (
+	"github.com/bytecodealliance/wasmtime-go/v12"
+
+	"go.uber.org/zap"
+
+	"github.com/ava-labs/avalanchego/utils/logging"
+
+	"github.com/ava-labs/hypersdk/state"
+	"github.com/ava-labs/hypersdk/x/programs/runtime"
+)
+
+// GlobalStorage is a global variable that holds program state.
```

Comment:
Can you add a TODO that this will be replaced with `storage` usage?

Reply:
this is now called `storage` and uses a db backend.